bika/lims/controlpanel/bika_instruments.py

```diff
@@ -108,17 +110,18 @@ def folderitems(self):
             items[x]['Model'] = obj.getModel()
 
             data = obj.getCertificateExpireDate()
-            if data == '':
-                items[x]['ExpiryDate'] = "No date avaliable"
+            if not data:
+                items[x]['ExpiryDate'] = _("No date set")
             else:
+                import pdb;pdb.set_trace()
```

Comment:
@Espurna @ramonski I'm not really sure that this line should be included in the merged PR.

Reply:
omg, I missed that line in my review 🙈 Thanks @rockfruit!